cmd/down.go

```diff
@@ -7,16 +7,23 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var downCmd = &cobra.Command{
-	Use: "down",
-	Short: "Deletes the pods of the specified docker compose services. " +
-		"If all docker compose services would be deleted then the Kubernetes services are also deleted.",
-	Long: "destroy all pods and services",
-	Run:  downCommand,
+func SetupDownCli() *cobra.Command {
+	var downCmd = &cobra.Command{
+		Use: "down",
+		Short: "Deletes the pods of the specified docker compose services. " +
+			"If all docker compose services would be deleted then the Kubernetes services are also deleted.",
+		Long: "destroy all pods and services",
+		Run:  downCommand,
+	}
+	return downCmd
 }
 
 func downCommand(cmd *cobra.Command, args []string) {
-	cfg, err := upOrDownCommandCommon(cmd, args)
+	file, err := getFileFlag(cmd)
```

Comment:
Should putShould move file back to `getCommandConfig`, so there's less duplication

Reply:
Done